src/components/pathway-progress/data/service.js

```diff
@@ -0,0 +1,81 @@
+// TODO: remove this once backend API is reeady and integrated.
```

Comment:
nit: I think we should add a link to the ticket where we are taking care of this integration